microprofile/microprofile-opentracing-1.0/src/test/java/org/wildfly/swarm/ts/microprofile/opentracing/v10/MicroProfileOpenTracing10IT.java

```diff
@@ -32,9 +32,16 @@
     public static void setUpJaeger() throws Exception {
         jaegerContainer = new Docker("jaeger", "jaegertracing/all-in-one:latest")
                 .waitForLogLine("\"Health Check state change\",\"status\":\"ready\"")
-                .port("6831:6831/udp") // default Jaeger agent
-                .port("16686:16686") // query service and UI
-                .port("14250:14250") // gRPC
+                 // Port documentation at https://www.jaegertracing.io/docs/latest/getting-started/
+                .port("5775:5775")
+                .port("5778:5778")
+                .port("9411:9411")
+                .port("6831:6831/udp")
+                .port("6832:6832")
+                .port("14250:14250")
+                .port("14267:14267") 
+                .port("14268:14268")
+                .port("16686:16686")
```

Comment:
There are 3 UDP ports, not just one.

Reply:
I think I removed them as I saw some failures on my tests these days. Updated.